README.md

```diff
@@ -60,6 +60,7 @@ type XhrOptions = String | {
     withCredentials: Boolean?,
     responseType: String?,
     beforeSend: Function?
+    qs: Any?
```

Comment:
@naugtur @jmeas yea you guys are right about that, have to agree, leave it as `Any?`